test/chalk.js

```diff
@@ -0,0 +1,104 @@
+import test from 'ava';
+import resolveFrom from 'resolve-from';
+
+// Spoof supports-color
+require.cache[resolveFrom(__dirname, 'supports-color')] = {
+	exports: {
+		level: 3,
+		hasBasic: true,
+		has256: true,
+		has16m: true
+	}
+};
```

Comment:
Potentially, but this is the only way to really guarantee what we want. I feel more comfortable with this since `supports-color`'s behavior might change.